tests/unit/test_charm_configure.py

```diff
@@ -47,8 +47,8 @@ def test_given_valid_config_when_config_changed_then_ca_certificate_is_pushed_to
         state_in = scenario.State(
             config={
                 "ca-common-name": "pizza.example.com",
-                "certificate-validity": 100,
-                "root-ca-validity": 200,
+                "certificate-validity-duration": "100d",
+                "root-ca-validity-duration": "200d",
```

Comment:
Should we duplicate one of (or all of?) these tests so that we don't accidentally break the deprecated config options in the future?

Reply:
I added a new singular test with this already, but maybe we won't need this if we decide to not deprecate the old endpoint.